src/sql/workbench/contrib/executionPlan/browser/executionPlan.ts

```diff
@@ -362,12 +368,50 @@ export class ExecutionPlan implements ISashLayoutProvider {
 			}
 		}
 
+		if (node.badges) {
+			diagramNode.badges = [];
+			for (let i = 0; i < node.badges.length; i++) {
+				diagramNode.badges.push(this.getBadgeTypeString(node.badges[i].type));
+			}
+		}
+
 		if (node.description) {
 			diagramNode.description = node.description;
 		}
 		return diagramNode;
 	}
 
+	private getBadgeTypeString(badgeType: BadgeType): {
+		type: string,
+		tooltip: string
+	} {
+		/**
+		 * TODO: Need to figure out if tooltip have to be removed. For now, they are empty
```

Comment:
If you're not using this then just remove it until you actually determine you need it.